string event_id
---
string timestamp
string[] person_ref
